go.mod

```diff
@@ -76,3 +76,5 @@ require (
 	google.golang.org/protobuf v1.33.0 // indirect
 	gopkg.in/ini.v1 v1.67.0 // indirect
 )
+
+replace github.com/PagerDuty/go-pagerduty => github.com/imjaroiswebdev/go-pagerduty-cli v0.0.0-20240517175334-7b299156d4a2
```

Comment:
This replacement should be dropped and deps updated onces https://github.com/PagerDuty/go-pagerduty/pull/528 get merged.

Reply:
Resolved in [b4e1e40](https://github.com/PagerDuty/terraform-provider-pagerduty/pull/875/commits/b4e1e4074e04e9a89cd10955927e1c4ccd16b7c3)